src/sass/getters/ranges.scss

```diff
@@ -52,3 +53,16 @@
 @function percent-range() {
```

Comment:
In theory we can delete `percent-range` right? Decimal-range is supposed to do all of this for us, I think. A percent-range in theory is just like, 0 to 100% or like decimal-range(0, 100) with a percent sign. Maybe we can add a third parameter that enables support for percentage-based values instead?  This isn’t something for you to worry about, but yeah, the idea is to have one getter that deals with all decimal-based incrementing.